### ISSUE TITLE
First Phase To-Do List

### ISSUE THREAD

**Issue by davidero1** [davidero1/Wokaholik#1]
Let's create the following pages first:
1. Get Started
2. Sign up
3. Log in
They should have accessibility links to each other.
